hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -195,20 +195,101 @@ msgstr "Support / Kontakt"
 
 #, python-format
 msgid ""
-"You're receiving this email because you requested a password reset for your "
-"user account at %(site_name)s."
+"You have ordered a new virtual machine!\n"
+"<br/>\n"
+"Your order of [%(vm_name)s] has been charged.<br/><br/>\n"
+"You can view your invoice by clicking the button below.<br/><br/>\n"
 msgstr ""
+"Du hast eine neue virtuelle Maschine bestellt!<br/>\n"
+"Deine Bestellung von [%(vm_name)s] wurde erhoben.<br/><br/>\n"
+"Um die Rechnung zu sehen, klicke auf den Button unten.<br/><br/>\n"
+
+msgid "View Invoice"
+msgstr "Zur Rechnung"
 
-msgid "Please go to the following page and choose a new password:"
+#, python-format
+msgid ""
+"You have ordered a new virtual machine!\n"
+"Your order of [%(vm_name)s] has been charged.\n"
+"You can view your invoice here.\n"
 msgstr ""
+"Du hast eine neue virtuelle Maschine bestellt!\n"
+"Deine Bestellung von [%(vm_name)s] wurde erhoben.\n"
+"Um die Rechnung zu sehen, klicke hier.\n"
 
-msgid "Thanks for using our site!"
+msgid "Password Reset"
+msgstr "Passwort zurücksetzen"
+
+#, python-format
+msgid ""
+"\n"
+"You're receiving this email because you requested a password reset for your "
+"user account at %(site_name)s.<br/>\n"
+"Please go to the following page and choose a new password: %(base_url)s"
+"%(password_reset_url)s<br/>\n"
+"If you didn't request a new password, ignore this e-mail.<br/>\n"
+"Thank you!\n"
 msgstr ""
+"\n"
+"Du erhälst diese E-Mail da du dein Passwort für deinen Account bei "
+"%(site_name)s zurücksetzen möchtest.<br/>\n"
+"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
+"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
+"dann ignoriere diese E-Mail.<br/>\n"
+"Dankeschön!\n"
+
+#, python-format
+msgid ""
+"You're receiving this email because you requested a password reset for your "
+"user account at %(site_name)s.\n"
+"Please go to the following page and choose a new password: %(base_url)s"
+"%(password_reset_url)s\n"
+"If you didn't request a new password, ignore this e-mail.\n"
+"Thank you!\n"
+msgstr ""
+"Du erhälst diese E-Mail da du dein Passwort für deinen Account bei "
+"%(site_name)s zurücksetzen möchtest.\n"
+"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
+"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
+"dann ignoriere diese E-Mail.\n"
+"Dankeschön!\n"
 
 #, python-format
-msgid "The %(site_name)s team"
+msgid ""
+"You're receiving this mail because your virtual machine [%(vm_name)s] has "
+"been cancelled.<br/>\n"
```

Comment:
I received vm name as [public-CentOS 7-17582] I think "public" part is not supposed to be there?

Reply:
Good catch @verysanghee I fixed that in the latest commit.